server/controller/opengraph.js

```diff
@@ -0,0 +1,173 @@
+const fs = require("fs");
+const resvg = require("@resvg/resvg-js").Resvg;
+const moment = require("moment-timezone");
+const tests = require("../controller/speedtests");
+const axios = require("axios");
+
+async function generateOpenGraphImage(req) {
+  const test = await tests.listStatistics(1);
+
+  if (!test.download.avg || !test.upload.avg || !test.ping.avg) {
+    throw new Error("Error fetching OpenGraph data");
+  }
+
+  const fontPath = "/assets/fonts/inter-v12-latin-regular.ttf";
+
+  const font =
+    process.env.NODE_ENV === "production"
+      ? (await axios.get(`${req.protocol}://${req.hostname}${fontPath}`)).data
+      : await fs.promises.readFile(`web/public${fontPath}`);
```

Comment:
```suggestion
      : await fs.promises.readFile(`client/public${fontPath}`);
```

Instead of using `web` here, please use the `client` folder instead. The `web` folder includes the landing page hosted at [myspeed.dev](https://myspeed.dev) while the `client` folder contains the files of the frontend